### ISSUE TITLE
Status bar item(s) per file/project

### ISSUE THREAD

**Issue by netizen-ais** [mehcode/linter-ui-plus#12]
I miss this and look forward to make the switch to this package. Now I miss this UI while using default package :smile:

**Comment by mehcode** [comment]
If you mean the numbers in the status bar that show total warnings and errors, that's now out in v0.3

**Comment by netizen-ais** [comment]
No, I mean, linter-ui-default has a setting to set the scope of the status bar item to file or project:
![spectacle sa5432](https://user-images.githubusercontent.com/3426685/28129643-57bb9910-6734-11e7-9323-f2f955a8fb1d.png)

**Comment by mehcode** [comment]
I'd like to avoid doing something for the sole reason that it was done in the previous project. I'm not saying we can't have this, just that I'd like to formalize the _why_.

Currently the Atom status bar _generally_ has project-level items on the outer-left and outer-right areas with file-level items on the inner-left and inner-right areas. This means that the current default of this package is incorrect with regards to that.

---

With that said, what are some reasons why you want this?

**Comment by netizen-ais** [comment]
It' for sure my fault, by I'm used to look at the status bar item before clicking my save and publish buttons. I also use [move-status-items](https://atom.io/packages/move-status-items) to have my status bar sorted to my confort.
